core/serialization/bindings.js

```diff
@@ -22,12 +22,15 @@ Serializer.defineSerializationUnit("bindings", function (serializer, object) {
 
         var output = {};
 
-        if (("serializable" in input) && !input.serializable)
+        if (("serializable" in input) && !input.serializable) {
             continue;
+        } else if (!input.sourceSyntax) {
```

Comment:
Is this line required do other changes on the PR or is it just a general fix?  I don't have a problem with including it. I just want to make sure I understand if and how it fits.